static/app/components/searchQueryBuilder/utils.tsx

```diff
@@ -107,3 +112,35 @@ export function formatFilterValue(token: TokenResult<Token.FILTER>): string {
       return token.value.text;
   }
 }
+
+export function shiftFocusToChild(
```

Comment:
This logic already existed, I just extracted it into this file so I could use it in the new paren component.